vshard/router/init.lua

```diff
@@ -1217,12 +1217,7 @@ local function master_search_set(router)
             end
         end
     else
-        -- Do not make users pay for what they do not use - when the search is
-        -- disabled for all replicasets, there should not be any fiber.
         log.info('Master auto search is disabled')
-        for _, replicaset in pairs(router.replicasets) do
```

Comment:
This was useless, because the loop goes other the newly created replicasets, which don't have `master_search` service at all. We don't need that `worker:remove_service` at all